components/sidebar/sidebar_category/sidebar_category_menu/index.ts

```diff
@@ -9,16 +9,22 @@ import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
 import {ActionFunc} from 'mattermost-redux/types/actions';
 import {GlobalState} from 'mattermost-redux/types/store';
 
+import {GlobalState as WebappGlobalState} from 'types/store';
```

Comment:
Instead of using `as`, you can just replace `GlobalState` with this everywhere in this file